build/openresty/wasmx/wasmx_repositories.bzl

```
@@ -60,7 +60,7 @@ filegroup(
 
 filegroup(
     name = "lib",
-    srcs = glob(["**/*.so", "**/*.dylib"]),
+    srcs = glob(["**/*.a", "**/*.dylib"]),
```

```suggestion
    srcs = glob(["**/*.a"]),
```
The dynamic library should not be necessary in macos either. I will test it locally.

---

Good call! I've committed this change. If something breaks in your local tests, feel free to revert.